Regarding issue #298

thomastthai opened the pull request:
Made the layout more intuitive.

brandyscarney commented on src/pages/speaker-list/speaker-list.html:
```diff
@@ -8,17 +8,14 @@
-        <button ion-button icon-only (click)="openContact(speaker)" clear item-right>
-          <ion-icon name="help-circle"></ion-icon>
-        </button>
+        <button ion-button clear  (click)="goToSpeakerDetail(speaker)">{{speaker.name}}</button>
```
Maybe this could be changed to where the entire item is clickable but without the arrow?

```
    <ion-card-header>
      <button ion-item detail-none (click)="goToSpeakerDetail(speaker)">
        <ion-avatar item-left>
          <img [src]="speaker.profilePic">
        </ion-avatar>
        {{speaker.name}}
      </button>
    </ion-card-header>
```

It's a bit confusing to click the text to navigate to info about the speaker. I wasn't expecting it when I first saw it.

brandyscarney commented on src/pages/speaker-list/speaker-list.html:
```diff
@@ -27,22 +24,29 @@
         <button ion-item *ngFor="let session of speaker.sessions" (click)="goToSessionDetail(session)">
           <h3>{{session.name}}</h3>
         </button>
-        <button ion-item (click)="goToSpeakerDetail(speaker)">
```
Can we leave this button in, please? :)

thomastthai commented:
*Hi Brandy. Thanks for reviewing my first ever PR! I am still learning. The
conference app has been helpful for me to learn Ionic 2. Thank you for all
the work!*

*As I learn and review the app, I look through the lens of the end user,
not the programmer. I strive to feel what the end user would feel as they
use the app. In showcasing Ionic's capabilities using the conference app,
this would be a wonderful opportunity to create an experience for the
developers as if they were end users of their potential product. A product
is only a product until it connects emotionally to the end user. When there
is that emotional connection, that product evolves to an experience. We
remember experiences, not so much product.*

*The class needs to be speaker-card in order for the following css to work.**
Yes. Good catch.*

*Maybe this could be changed to where the entire item is clickable but
without the arrow?** Making the avatar and name one clickable item would
work make it even better. The idea was to make it interactive. By arrow do
you mean the Share button? I thought the original intent for Share button
was to allow people to share about the presenter and what he/she is
presenting. Personally, I would clean it up further and move the Tweet
button into the Share button menu and move the Share button to the left
corner. I replaced the icon with the arrow because that usually represents
sharing?*

*It's a bit confusing to click the text to navigate to info about the
speaker. I wasn't expecting it when I first saw it.** By text do you mean
the name of the presenter? The event text below is clicakable and takes the
user to detailed info about the events. That same throught process should
be extensed to the presenter's name to keep the user experience consistent.*

*Can we leave this button in, please? :) **The original question mark
button at the upper right corner that takes a user to details about the
speaker?  From a user experience, what value does that button bring?  The
value that it bring is taking the user to the presenter detail page. That
was accomplished in a more intuitive way by allowing the user to click on
the avatar and presenter's name. What cost is it to the user experience?
IMHO, it clutters up that top part and takes focus way from the information
being presented. Less is more.*

Thanks for sharing your thoughts and considering mine. I have a lot more
thoughts with regard to showcasing the features of Ionic to create
emotional connections with end user's experience. I will update as I learn
more about programming.

On Nov 9, 2016 12:09 PM, "Brandy Carney" <notifications@github.com> wrote:

> *@brandyscarney* commented on this pull request.
>
> Thanks for the PR! I added some comments for some changes. Let me know
> your thoughts. :)
> ------------------------------
>
> In src/pages/speaker-list/speaker-list.html
> <https://github.com/driftyco/ionic-conference-app/pull/306#pullrequestreview-7884806>
> :
>
> > @@ -8,17 +8,14 @@
>  </ion-header>
>
>  <ion-content class="outer-content speaker-list">
> -  <ion-card *ngFor="let speaker of speakers" class="speaker-card">
> +  <ion-card *ngFor="let speaker of speakers" class="speaker">
>
> The class needs to be speaker-card in order for the following css to
> work: https://github.com/driftyco/ionic-conference-app/blob/
> master/src/pages/speaker-list/speaker-list.scss#L8-L23
> ------------------------------
>
> In src/pages/speaker-list/speaker-list.html
> <https://github.com/driftyco/ionic-conference-app/pull/306#pullrequestreview-7884806>
> :
>
> >          </ion-avatar>
> -        {{speaker.name}}
> -        <button ion-button icon-only (click)="openContact(speaker)" clear item-right>
> -          <ion-icon name="help-circle"></ion-icon>
> -        </button>
> +        <button ion-button clear  (click)="goToSpeakerDetail(speaker)">{{speaker.name}}</button>
>
> Maybe this could be changed to where the entire item is clickable but
> without the arrow?
>
>     <ion-card-header>
>       <button ion-item detail-none (click)="goToSpeakerDetail(speaker)">
>         <ion-avatar item-left>
>           <img [src]="speaker.profilePic">
>         </ion-avatar>
>         {{speaker.name}}
>       </button>
>     </ion-card-header>
>
> It's a bit confusing to click the text to navigate to info about the
> speaker. I wasn't expecting it when I first saw it.
> ------------------------------
>
> In src/pages/speaker-list/speaker-list.html
> <https://github.com/driftyco/ionic-conference-app/pull/306#pullrequestreview-7884806>
> :
>
> > @@ -27,22 +24,29 @@
>          <button ion-item *ngFor="let session of speaker.sessions" (click)="goToSessionDetail(session)">
>            <h3>{{session.name}}</h3>
>          </button>
> -        <button ion-item (click)="goToSpeakerDetail(speaker)">
>
> Can we leave this button in, please? :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/driftyco/ionic-conference-app/pull/306#pullrequestreview-7884806>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APxEZY1nurtxA7QcjyA_yx1LBE6UldIdks5q8ihvgaJpZM4KthOc>
> .
>

brandyscarney commented:
I went ahead and made the changes and squashed them into one commit from you. Sorry for not being descriptive enough! :) Thanks! https://github.com/driftyco/ionic-conference-app/commit/31a60288fa55bc481d6e4b60f79e68c93a14c92d

thomastthai commented:
Thank you Brandy. I am curious how extensive do you all want to take the
conference app?

On Nov 10, 2016 11:08 AM, "Brandy Carney" <notifications@github.com> wrote:

> I went ahead and made the changes and squashed them into one commit from
> you. Sorry for not being descriptive enough! :) Thanks! 31a6028
> <https://github.com/driftyco/ionic-conference-app/commit/31a60288fa55bc481d6e4b60f79e68c93a14c92d>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/driftyco/ionic-conference-app/pull/306#issuecomment-259778584>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APxEZQz9ML-Hoj9I2tbIKQ875dkKTXr1ks5q82uMgaJpZM4KthOc>
> .
>

brandyscarney commented:
@thomastthai Well we have a lot of changes that will be added here: https://github.com/driftyco/ionic-conference-app/pull/307

Our goal for the conference app is to showcase a complicated example of what Ionic can do, including the different components available, navigation stacks, styling, etc. We eventually want to integrate Ionic Cloud, as well. :)

thomastthai commented:
Thank you Brandy.

On Nov 10, 2016 12:01 PM, "Brandy Carney" <notifications@github.com> wrote:

> @thomastthai <https://github.com/thomastthai> Well we have a lot of
> changes that will be added here: #307
> <https://github.com/driftyco/ionic-conference-app/pull/307>
>
> Our goal for the conference app is to showcase a complicated example of
> what Ionic can do, including the different components available, navigation
> stacks, styling, etc. We eventually want to integrate Ionic Cloud, as well.
> :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/driftyco/ionic-conference-app/pull/306#issuecomment-259792065>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APxEZQbHlX8Tu2LwQKGWaaLARRiC-cxfks5q83gBgaJpZM4KthOc>
> .
>

thomastthai commented:
Brandy,

I'm still learning GitHub. What is the difference between the Master branch
and the Typescript branch? Which should I be pulling? I last pulled the
master branch and when I did a comparison there was very little change.
However, when I compared my branch with the Typescript branch, there are
changes going back several months.

I was going to submit another pull request to correct this error in
speaker-list.scss. When width is set to 60% it looks fine on a cell phone
but not on a progressive web page. It becomes a smaller box for each card
at 60%.

  @media (min-width: 600px) {
    .speaker-card {
      width: 100%; // <--- was 60%
      margin: 10px auto;
    }
  }

On Thu, Nov 10, 2016 at 12:26 PM, Thomas Thai <thomas.t.thai@gmail.com>
wrote:

> Thank you Brandy.
>
> On Nov 10, 2016 12:01 PM, "Brandy Carney" <notifications@github.com>
> wrote:
>
>> @thomastthai <https://github.com/thomastthai> Well we have a lot of
>> changes that will be added here: #307
>> <https://github.com/driftyco/ionic-conference-app/pull/307>
>>
>> Our goal for the conference app is to showcase a complicated example of
>> what Ionic can do, including the different components available, navigation
>> stacks, styling, etc. We eventually want to integrate Ionic Cloud, as well.
>> :)
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/driftyco/ionic-conference-app/pull/306#issuecomment-259792065>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/APxEZQbHlX8Tu2LwQKGWaaLARRiC-cxfks5q83gBgaJpZM4KthOc>
>> .
>>
>

brandyscarney commented:
Hey @thomastthai, the typescript branch is an old branch that should probably be deleted. The master branch is the "main" branch that all PRs should be forked from. :)

thomastthai commented:
Thank you.

On Nov 21, 2016 12:43 PM, "Brandy Carney" <notifications@github.com> wrote:

> Hey @thomastthai <https://github.com/thomastthai>, the typescript branch
> is an old branch that should probably be deleted. The master branch is the
> "main" branch that all PRs should be forked from. :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/driftyco/ionic-conference-app/pull/306#issuecomment-262061221>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APxEZReL4Y-2uDnTcRubmZcwjvbN4tyIks5rAgJlgaJpZM4KthOc>
> .
>
